src/x86_64/unwind_rule.rs

```diff
@@ -19,6 +20,120 @@ pub enum UnwindRuleX86_64 {
     },
     /// (sp, bp) = (bp + 16, *bp)
     UseFramePointer,
```

Comment:
Both Clang and MSVC will use "base pointers", a generalization of frame pointer, when both alloca and stack realignment are involved. This consists a small but often measurable amount of sampled stacks. My fork originally handled that case, and I still think we should have a rule for that.

https://github.com/ishitatsuyuki/framehop/blob/4b6f62a8b8a046a06c56db534dd79f10b1f32472/src/x86_64/unwind_rule.rs#L22-L26